homeassistant/components/swiss_public_transport/sensor.py

```diff
@@ -101,14 +100,24 @@ class SwissPublicTransportSensor(
 
     _attr_attribution = "Data provided by transport.opendata.ch"
     _attr_icon = "mdi:bus"
+    _attr_has_entity_name = True
+    _attr_translation_key = "departure"
 
     def __init__(
-        self, coordinator: SwissPublicTransportDataUpdateCoordinator, name: str
+        self,
+        coordinator: SwissPublicTransportDataUpdateCoordinator,
+        unique_id: str,
     ) -> None:
         """Initialize the sensor."""
         super().__init__(coordinator)
         self._coordinator = coordinator
-        self._attr_name = name
+        self._attr_name = "Departure"
```

Comment:
```suggestion
```

Reply:
Is this not needed when translation key is provided?

Reply:
Nope. Probably when locally testing you don't see the name of the entity, but this is because you probably didnt generate translations. (HA uses the `translations/en.json` at runtime and strings.json at compile time)

Reply:
Yes, had already a look at "local" translations. I am using dev_containers, therefore it is quite fast and easy to recompile it :)